internal/backend/db/dbgorm/vars.go

```diff
@@ -0,0 +1,116 @@
+package dbgorm
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	"gorm.io/gorm/clause"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/db/dbgorm/scheme"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkerrors"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+func (db *gormdb) SetVars(ctx context.Context, vars []sdktypes.Var) error {
+	if i, err := kittehs.ValidateList(vars, func(_ int, v sdktypes.Var) error {
+		return v.Strict()
+	}); err != nil {
+		return fmt.Errorf("#%d: %w", i, err)
```

Comment:
let's add a string to make this error more human-friendly?
fmt.Errorf("setVars: #%d: %w", i, err)

Reply:
i think that adding the `setVars` perfix is wrong, the calling function should add it.

Reply:
then we need to ensure that calling function is doing this, which is not the case :)
that's why I suggest to add it here at least